nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -208,7 +315,8 @@ IniGraphInterpreter::deserialize(const std::string &in) {
         layer = section2layer<PlainLayer>(ini, sec_name, app_context);
       } else {
         /// @todo deprecate this as well with #1072
```

Comment:
you can consider adding #1072 here. It might simplify your work :)

Reply:
It is true 👍, I'll do it in the later PR